File: backend/src/main/java/Manager/Restaurant/mai/dto/ReorderRequestDTO.java
package Manager.Restaurant.mai.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ReorderRequestDTO {
    private Long orderId;
    private Double latitude;
    private Double longitude;
}

File: frontend/36Food/app/src/main/java/com/example/a36food/data/dto/ReorderRequestDTO.kt
package com.example.a36food.data.dto

import com.google.gson.annotations.SerializedName

data class ReorderRequestDTO(
    @SerializedName("orderId")
    val orderId: Long,

    @SerializedName("latitude")
    val latitude: Double,

    @SerializedName("longitude")
    val longitude: Double
)
